Description: Make bindgen and pkg-config features empty as they are "mandatory"
 We use Debian packaged libzstd, so these features are "chosen" anyway.
Last-Update: 2022-12-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -47,7 +47,7 @@
 
 [features]
 arrays = []
-bindgen = ["zstd-sys/bindgen"]
+bindgen = []
 debug = ["zstd-sys/debug"]
 default = [
     "legacy",
@@ -58,7 +58,7 @@
 experimental = ["zstd-sys/experimental"]
 legacy = ["zstd-sys/legacy"]
 no_asm = ["zstd-sys/no_asm"]
-pkg-config = ["zstd-sys/pkg-config"]
+pkg-config = []
 std = ["zstd-sys/std"]
 thin = ["zstd-sys/thin"]
 zdict_builder = ["zstd-sys/zdict_builder"]
